.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "8"
+  - "7"
+  - "6"
+  - "5"
```

Comment:
Node.js 5 and 7 have reached their end of life. We should add `9` if we want to test any of the odds. It also might be good to add a failure ignore for the odd versions:

https://github.com/nodejs/Release

https://github.com/twolfson/gulp.spritesmith/blob/6.9.0/.travis.yml